src/OrbitGl/TimeGraph.h

```diff
@@ -159,6 +158,7 @@ class TimeGraph final : public orbit_gl::CaptureViewElement,
  protected:
   void PrepareBatcherAndUpdatePrimitives(PickingMode picking_mode);
   void DoUpdateLayout() override;
+  void UpdateChildrenPosAndSize();
```

Comment:
Nit: `UpdateChildrenPosAndContainerSize` would be more accurate as you're not touching the size of any children except the track container.